setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="vumi_twilio_api",
```

Comment:
Also wondering whether using Twilio in the name might cause trademark issues? If so, maybe `vxtwinio`? :)